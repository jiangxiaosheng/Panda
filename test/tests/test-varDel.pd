var a: int
var b: float
var c: string
var d: boolean

func main() {
	a = 1
	b = 2.0
	c =  "hello"
	d = false
}