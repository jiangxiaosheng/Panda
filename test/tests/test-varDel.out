

Sementically checked program: 

var a: int(int : 0)
var b: float(float : 0.)
var c: string(string : )
var d: bool(bool : false)

void main() {
(int : a = (int : 1))
(float : b = (float : 2.))
(string : c = (string : hello))
(bool : d = (bool : false))
}

